src/components/02-components/data-table/data-table.data.tsx

```diff
@@ -228,3 +228,36 @@ export const staffFilterModel: GridFilterModel = {
   items: [{ field: 'col3', operator: 'contains', value: 'Education' }],
   logicOperator: GridLogicOperator.Or,
 };
+
+export const enrolledStudentsTableRows = [
+  {
+    id: 1,
+    firstName: 'Emily',
+    lastName: 'Parker',
+    lastlogin: '09/12/2023 - 10:10am',
+    experiences: '#2 - Community Roots Garden',
+  },
+  {
+    id: 2,
+    firstName: 'Mathew',
+    lastName: 'Ross',
+    lastlogin: '03/28/2023 - 11:16am',
+    experiences: '#1 - Community Roots Garden',
+  },
+];
+
+export const enrolledStudentsTableColumns = [
+  { field: 'firstName', headerName: 'Name', width: 200 },
```

Comment:
Open question with Judy on changing this to "First Name": https://fourkitchens.slack.com/archives/C7Q8W0EAK/p1695142777188469